Shared/Prefs.swift

```diff
@@ -120,6 +120,7 @@ public struct PrefsKeys {
     public static let KeyInactiveTabsFirstTimeRun = "KeyInactiveTabsFirstTimeRunKey"
     public static let KeyTabDisplayOrder = "KeyTabDisplayOrderKey"
     public static let TabMigrationKey = "TabMigrationKey"
+    public static let tabSyncEnabled = "sync.engine.tabs.enabled"
```

Comment:
We should follow the convention of all the other keys here. 
```suggestion
    public static let TabSyncEnabled = "sync.engine.tabs.enabled"
```